src/service/searchService.ts

```diff
@@ -78,8 +78,9 @@ export class SearchService {
             }
 
             for (const brandPid of brandPids) {
-                const { data: { children: seriesList } }: { data: IPlayerChilrenResponse } = await axios.get(`https://www.bbc.co.uk/programmes/${encodeURIComponent(brandPid)}/children.json?limit=100`);
-                const episodes = (await Promise.all(seriesList.programmes.filter(({ type, title }) => type == 'series' && !title.toLocaleLowerCase().includes('special')).map(({ pid }) => episodeCacheService.getSeriesEpisodes(pid)))).flat();
+                const { data: { children: seriesList } }: { data: IPlayerChildrenResponse } = await axios.get(`https://www.bbc.co.uk/programmes/${encodeURIComponent(brandPid)}/children.json?limit=100`);
```

Comment:
Been looking into the API used for Native Search, may be able to use it to get results similar to the search in iPlayer. It has availability on the results too, which is nice. Future PR if I can get it to work.